src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+let employeesList = [...document.querySelectorAll('li')];
+
+const helper = (list) => {
+  return list.map((x) => {
+    return {
+      salary: Number(x.dataset.salary.split(',').join('').slice(1)),
+      element: x,
+    };
+  });
+};
+
+employeesList = helper(employeesList);
+
+const sortList = (list) => {
+  return list.sort((a, b) => b.salary - a.salary);
+};
+
+employeesList = sortList(employeesList);
+
+const parentt = document.querySelector('ul');
+
+parent.innerHTML = '';
```

Comment:
The variable `parent` is used here, but it is not defined in the code. You probably meant to use `parentt`, which is defined in line 22. Correct the typo in the variable name to ensure consistency.